src/main/java/ChattyCharlie.java

```diff
@@ -0,0 +1,189 @@
+            //account for the item
+            size++;
+        }
+
+        //To mark
+        public void mark(int index) {
+            if (index >= 0 && index < size) {
+                list[index].markTask();
+                int remainingTask = countUnmarkedTasks();
+                System.out.println("        Well Done! 1 task down, " + remainingTask + " to go.");
+                System.out.println("        [" + list[index].getMarkedStatus() + "] " + list[index].description);
+            } else {
+                System.out.println("        Invalid task number.");
+            }
+        }
+
+        //To unmark
+        public void unmark(int index) {
+            if (index >= 0 && index < size) {
+                list[index].unmarkTask();
+                int remainingTask = countUnmarkedTasks();
+                System.out.println("        Hmmm, not quite done yet, " + remainingTask + " to go.");
+                System.out.println("        [" + list[index].getMarkedStatus() + "] " + list[index].description);
+            } else {
+                System.out.println("        Invalid task number.");
+            }
+        }
+
+        //To print list
+        public void toPrintList() {
+            //print all
+            int remainingTask = countUnmarkedTasks();
+            System.out.println("ToDo List:");
+            System.out.println("pending Task: " + remainingTask);
+            for (int i = 0; i < size; i++) {
+                int number = i+1;
+                System.out.println("        " + number + ".[" +list[i].getMarkedStatus() + "] " + list[i].description);
+            }
+        }
+
+        // Method to count how many tasks are unmarked
+        public int countUnmarkedTasks() {
+            int count = 0;
+            for (int i = 0; i < size; i++) {
+                if (!list[i].isDone) {
+                    count++;
+                }
+            }
+            return count;
+        }
+    }
+
+    //MAIN ALGO
+    public static void toDoMaker() { //Echo as a function
+        String line;
+        String you = "User: ";
+
+        //make the scanner
+        Scanner in = new Scanner(System.in);
+
+        //create an instance of list class
+        List toDo = new List();
+
+        //accept an insert
+        while (true) {
+            System.out.print(you);
+            line = in.nextLine();
+
+            //if the line contains bye, it signals the end
+            if (line.contains("Bye") || line.contains("bye")) {
+                break;
+            }
+            //add or print
+            if (line.equals("list")) {
+                toDo.toPrintList();
+            } else if (line.startsWith("mark ")) {
+                //trim to get the task name
+                String taskDescription = line.substring(5).trim();
+                boolean found = false;
+
+                //loop to find
+                for(int index = 0; index < toDo.size; index++)
+                {
```

Comment:
Not following loop layout coding standard

Reply:
Sorry I dont understand what you mean, is this not the standard? 

There are curly brackets, can I clarfiy what fo you mean